bower.json

```diff
@@ -2,9 +2,6 @@
   "name": "PodTestPrototype",
   "version": "1.0.0",
   "homepage": "https://github.com/urbanpod/PodTestPrototype",
-  "authors": [
```

Comment:
Yeah also b/c bower.json. I went in to clean it and thought, "hmmm when the team gets bigger I don't want to keep the list here updated"